packages/contracts/contracts/GenArt721CoreV3.sol

```diff
@@ -1061,6 +1062,28 @@ contract GenArt721CoreV3 is
         emit ProjectUpdated(_projectId, FIELD_PROJECT_SCRIPT);
     }
 
+    /**
+     * @notice Adds a pre-compressed script to project `_projectId`.
+     * @param _projectId Project to be updated.
+     * @param _compressedScript Pre-compressed script to be added.
+     * Required to be non-empty, but no further validation is performed.
+     * @dev  A script can be pre-compressed by calling `getProjectScriptCompressed`.
+     */
+    function addProjectScriptCompressed(
+        uint256 _projectId,
+        bytes memory _compressedScript
+    ) external {
+        _onlyUnlocked(_projectId);
+        _onlyArtistOrAdminACL(_projectId, this.addProjectScript.selector);
```

Comment:
update this to `this.addProjectScriptCompressed.selector`

Reply:
ah great catch, ty!